### ISSUE TITLE
Leave scroll position at bottom of Location List

### ISSUE THREAD

**Issue by devth** [tpope/vim-dispatch#101]
After running a `:Dispatch`, is it possible to put scroll position at the bottom of the Location List? It's jarring to watch a mvn build progress, then jump to the top.

**Comment by watsoncj** [comment]
Agreed. This is a very common use case to scroll to the bottom to see the build result.

**Comment by tpope** [comment]
I think I would be okay with this in the case that there were no specific errors matched, just a capture of all output.
